### DISCUSSION TITLE
How to Deploy monitoring services on Kubernetes?

### DISCUSSION THREAD

**Discussion by kinoxyz1** [milvus-io/milvus#34808] [Q&A and General discussion]
My Milvus cluster is deployed using Milvus operator. I see only helm-related [documents](https://milvus.io/docs/monitor.md#Deploy-monitoring-services-on-Kubernetes) on deploying monitoring services. How do I configure monitoring services for a cluster deployed by Milvus operator? 

If possible, I would like a document, thank you.

**Comment by xiaofan-luan** [comment]
@LoveEachDay 
could you help on this
